### ISSUE TITLE
logits to be able to save probe data in a single file.

### ISSUE THREAD

**Issue by qualiteger** [qualiteg/ChatStream#37]
・logitsファイルが大量に生まれるため
・mock_def で切り替えられる
